contracts/priv/unity.yml

```diff
@@ -50,6 +50,67 @@ paths:
         default:
           description: Unexpected error
           $ref: '#/components/responses/ServerError'
+  /accounts:
+    get:
+      operationId: GetAccounts
+      tags:
+        - Accounts
+      summary: Get the list of accounts for the current user
+      responses:
+        '200':
+          description: Accounts
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  properties:
+                    id:
+                      type: number
+                      description: account id in quartz
+                    isActive:
+                      type: boolean
+                      description: is this the currently active account in the session?
+                    isDefault:
+                      type: boolean
+                      description: is this the user's default account?
+                    name:
+                      type: string
+                      description: name of the account
+                  required:
+                    - id
+                    - isActive
+                    - isDefault
+                    - name
+        '401':
+          description: Unauthorized
+          $ref: '#/components/responses/ServerError'
+        default:
+          description: Unexpected error
+          $ref: '#/components/responses/ServerError'
+  /accounts/default:
+    put:
+      operationId: PutDefaultAccount
+      tags:
+        - Accounts
+      requestBody:
```

Comment:
out of curiosity, why put the id in a request body and not as part of the url? e.g.

```
accounts/default/account_id
```

Reply:
No real reason other than it looked kinda weird to me. I also considered `accounts/:account_id/default`, but that also felt weird.

If putting it the URL is better for you, I'm happy to change it. Note, though, that because we enforce JSON content type, you'll have to pass an empty body with that.

Let me know your preference and I'll either update or merge.

Reply:
no preference, just curious. I think enforcing a JSON content type and needing a body means this is fine :)